asv/benchmarks/cwt_benchmarks.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import pywt
+
+
+class CwtTimeSuiteBase(object):
+    """
+    Set-up for CWT timing.
+    """
+    params = ([32, 128, 512],
+              ['cmor', 'cgau4', 'fbsp', 'gaus4', 'mexh', 'morl', 'shan'],
+              [16, 64, 256])
+    param_names = ('n', 'wavelet', 'max_scale')
+
+    def setup(self, n, wavelet, max_scale):
+        try:
+            from pywt import cwt
+        except ImportError:
+            raise NotImplementedError("cwt not available")
+        self.data = np.ones(n, dtype='float')
+        self.wavelet = pywt.ContinuousWavelet(wavelet)
+        self.scales = np.arange(1, max_scale+1)
+
+
+class CwtTimeSuite(CwtTimeSuiteBase):
+    def time_cwt(self, n, wavelet, max_scale):
+        pywt.cwt(self.data, self.scales, self.wavelet)
```

Comment:
In this benchmark, self.scales is a range while max_scale is an integer

for `self.wavelet` vs. `wavelet` I think it depends which code path you want to benchmark.  For small sizes, the benchmark runs a bit faster if the input is already a Wavelet object, but for larger transforms the overhead of converting a string to a Wavelet object is negligable.